### ISSUE TITLE
Runtime Error: index out of range [2] with length 1 when running Search method on Netnaija Engine

### ISSUE THREAD

**Issue by derhnyel** [Go-phie/gophie#50]
Logs in verbose mode

```
go run .\main.go search Spider Man -v
time="2022-07-31T03:15:16+01:00" level=debug msg="Switching to ChromeDpTransport"
time="2022-07-31T03:15:18+01:00" level=debug msg="Visiting https://www.thenetnaija.com/search?folder=videos&t=Spider+Man"
time="2022-07-31T03:15:18+01:00" level=debug msg="Set Headers for page https://www.thenetnaija.com/search?folder=videos&t=Spider+Man"
time="2022-07-31T03:15:19+01:00" level=debug msg="Set Headers for page https://www.thenetnaija.com/search?folder=videos&t=Spider+Man"
time="2022-07-31T03:15:19+01:00" level=debug msg="Successfully retrieved body"
time="2022-07-31T03:15:19+01:00" level=debug msg="Done https://www.thenetnaija.com/search?folder=videos&t=Spider+Man"
time="2022-07-31T03:15:19+01:00" level=debug msg="Retrieving Download Link https://www.thenetnaija.net/videos/series/11602-marvels-spider-man/season-3/episode-4\n"
time="2022-07-31T03:15:19+01:00" level=debug msg="Set Headers for page https://www.thenetnaija.net/videos/series/11602-marvels-spider-man/season-3/episode-4"
time="2022-07-31T03:15:20+01:00" level=debug msg="Set Headers for page https://www.thenetnaija.net/videos/series/11602-marvels-spider-man/season-3/episode-4"
time="2022-07-31T03:15:20+01:00" level=debug msg="Successfully retrieved body"
time="2022-07-31T03:15:20+01:00" level=debug msg="Retrieved Download Link https://www.thenetnaija.net/videos/series/11602-marvels-spider-man/season-3/episode-4\n"
time="2022-07-31T03:15:20+01:00" level=debug msg="Retrieving Download Link https://www.thenetnaija.net/videos/series/11602-marvels-spider-man/season-3/episode-4/download\n"
time="2022-07-31T03:15:20+01:00" level=debug msg="Set Headers for page https://www.thenetnaija.net/videos/series/11602-marvels-spider-man/season-3/episode-4/download"
time="2022-07-31T03:15:22+01:00" level=debug msg="Set Headers for page https://www.thenetnaija.net/videos/series/11602-marvels-spider-man/season-3/episode-4/download"
time="2022-07-31T03:15:22+01:00" level=debug msg="Successfully retrieved body"
time="2022-07-31T03:15:22+01:00" level=debug msg="Retrieved Download Link https://www.thenetnaija.net/videos/series/11602-marvels-spider-man/season-3/episode-4/download\n"
time="2022-07-31T03:15:23+01:00" level=debug msg="Retrieving Download Link https://www.thenetnaija.net/videos/series/11602-marvels-spider-man/season-3/episode-5\n"
time="2022-07-31T03:15:23+01:00" level=debug msg="Set Headers for page https://www.thenetnaija.net/videos/series/11602-marvels-spider-man/season-3/episode-5"
time="2022-07-31T03:15:23+01:00" level=debug msg="Set Headers for page https://www.thenetnaija.net/videos/series/11602-marvels-spider-man/season-3/episode-5"
time="2022-07-31T03:15:23+01:00" level=debug msg="Successfully retrieved body"
time="2022-07-31T03:15:23+01:00" level=debug msg="Retrieved Download Link https://www.thenetnaija.net/videos/series/11602-marvels-spider-man/season-3/episode-5\n"
time="2022-07-31T03:15:23+01:00" level=debug msg="Retrieving Download Link https://www.thenetnaija.net/videos/series/11602-marvels-spider-man/season-3/episode-5/download\n"
time="2022-07-31T03:15:23+01:00" level=debug msg="Set Headers for page https://www.thenetnaija.net/videos/series/11602-marvels-spider-man/season-3/episode-5/download"
time="2022-07-31T03:15:25+01:00" level=debug msg="Set Headers for page https://www.thenetnaija.net/videos/series/11602-marvels-spider-man/season-3/episode-5/download"
time="2022-07-31T03:15:25+01:00" level=debug msg="Successfully retrieved body"
time="2022-07-31T03:15:25+01:00" level=debug msg="Retrieved Download Link https://www.thenetnaija.net/videos/series/11602-marvels-spider-man/season-3/episode-5/download\n"
time="2022-07-31T03:15:25+01:00" level=debug msg="Retrieving Download Link https://www.thenetnaija.net/videos/series/11602-marvels-spider-man/season-3/episode-2\n"
time="2022-07-31T03:15:25+01:00" level=debug msg="Set Headers for page https://www.thenetnaija.net/videos/series/11602-marvels-spider-man/season-3/episode-2"
time="2022-07-31T03:15:26+01:00" level=debug msg="Set Headers for page https://www.thenetnaija.net/videos/series/11602-marvels-spider-man/season-3/episode-2"
time="2022-07-31T03:15:26+01:00" level=debug msg="Successfully retrieved body"
time="2022-07-31T03:15:26+01:00" level=debug msg="Retrieved Download Link https://www.thenetnaija.net/videos/series/11602-marvels-spider-man/season-3/episode-2\n"
time="2022-07-31T03:15:26+01:00" level=debug msg="Retrieving Download Link https://www.thenetnaija.net/videos/series/11602-marvels-spider-man/season-3/episode-2/download\n"
time="2022-07-31T03:15:26+01:00" level=debug msg="Set Headers for page https://www.thenetnaija.net/videos/series/11602-marvels-spider-man/season-3/episode-2/download"
time="2022-07-31T03:15:27+01:00" level=debug msg="Set Headers for page https://www.thenetnaija.net/videos/series/11602-marvels-spider-man/season-3/episode-2/download"
time="2022-07-31T03:15:27+01:00" level=debug msg="Successfully retrieved body"
time="2022-07-31T03:15:27+01:00" level=debug msg="Retrieved Download Link https://www.thenetnaija.net/videos/series/11602-marvels-spider-man/season-3/episode-2/download\n"
time="2022-07-31T03:15:28+01:00" level=debug msg="Retrieving Download Link https://www.thenetnaija.net/videos/series/11602-marvels-spider-man/season-3/episode-3\n"
time="2022-07-31T03:15:28+01:00" level=debug msg="Set Headers for page https://www.thenetnaija.net/videos/series/11602-marvels-spider-man/season-3/episode-3"
time="2022-07-31T03:15:28+01:00" level=debug msg="Set Headers for page https://www.thenetnaija.net/videos/series/11602-marvels-spider-man/season-3/episode-3"
time="2022-07-31T03:15:28+01:00" level=debug msg="Successfully retrieved body"
time="2022-07-31T03:15:28+01:00" level=debug msg="Retrieved Download Link https://www.thenetnaija.net/videos/series/11602-marvels-spider-man/season-3/episode-3\n"
time="2022-07-31T03:15:28+01:00" level=debug msg="Retrieving Download Link https://www.thenetnaija.net/videos/series/11602-marvels-spider-man/season-3/episode-3/download\n"
time="2022-07-31T03:15:28+01:00" level=debug msg="Set Headers for page https://www.thenetnaija.net/videos/series/11602-marvels-spider-man/season-3/episode-3/download"
time="2022-07-31T03:15:30+01:00" level=debug msg="Set Headers for page https://www.thenetnaija.net/videos/series/11602-marvels-spider-man/season-3/episode-3/download"
time="2022-07-31T03:15:30+01:00" level=debug msg="Successfully retrieved body"
time="2022-07-31T03:15:30+01:00" level=debug msg="Retrieved Download Link https://www.thenetnaija.net/videos/series/11602-marvels-spider-man/season-3/episode-3/download\n"
time="2022-07-31T03:15:30+01:00" level=debug msg="Retrieving Download Link https://www.thenetnaija.net/videos/series/11602-marvels-spider-man/season-3/episode-1\n"
time="2022-07-31T03:15:30+01:00" level=debug msg="Set Headers for page https://www.thenetnaija.net/videos/series/11602-marvels-spider-man/season-3/episode-1"
time="2022-07-31T03:15:30+01:00" level=debug msg="Set Headers for page https://www.thenetnaija.net/videos/series/11602-marvels-spider-man/season-3/episode-1"
time="2022-07-31T03:15:30+01:00" level=debug msg="Successfully retrieved body"
time="2022-07-31T03:15:30+01:00" level=debug msg="Retrieved Download Link https://www.thenetnaija.net/videos/series/11602-marvels-spider-man/season-3/episode-1\n"
time="2022-07-31T03:15:30+01:00" level=debug msg="Retrieving Download Link https://www.thenetnaija.net/videos/series/11602-marvels-spider-man/season-3/episode-1/download\n"
time="2022-07-31T03:15:30+01:00" level=debug msg="Set Headers for page https://www.thenetnaija.net/videos/series/11602-marvels-spider-man/season-3/episode-1/download"
time="2022-07-31T03:15:32+01:00" level=debug msg="Set Headers for page https://www.thenetnaija.net/videos/series/11602-marvels-spider-man/season-3/episode-1/download"
time="2022-07-31T03:15:32+01:00" level=debug msg="Successfully retrieved body"
time="2022-07-31T03:15:32+01:00" level=debug msg="Retrieved Download Link https://www.thenetnaija.net/videos/series/11602-marvels-spider-man/season-3/episode-1/download\n"
time="2022-07-31T03:15:32+01:00" level=debug msg="Retrieving Download Link https://www.thenetnaija.net/videos/series/11602-marvels-spider-man/season-3/episode-6\n"
time="2022-07-31T03:15:32+01:00" level=debug msg="Set Headers for page https://www.thenetnaija.net/videos/series/11602-marvels-spider-man/season-3/episode-6"
time="2022-07-31T03:15:33+01:00" level=debug msg="Set Headers for page https://www.thenetnaija.net/videos/series/11602-marvels-spider-man/season-3/episode-6"
time="2022-07-31T03:15:33+01:00" level=debug msg="Successfully retrieved body"
time="2022-07-31T03:15:33+01:00" level=debug msg="Retrieved Download Link https://www.thenetnaija.net/videos/series/11602-marvels-spider-man/season-3/episode-6\n"
time="2022-07-31T03:15:33+01:00" level=debug msg="Retrieving Download Link https://www.thenetnaija.net/videos/series/11602-marvels-spider-man/season-3/episode-6/download\n"
time="2022-07-31T03:15:33+01:00" level=debug msg="Set Headers for page https://www.thenetnaija.net/videos/series/11602-marvels-spider-man/season-3/episode-6/download"
time="2022-07-31T03:15:34+01:00" level=debug msg="Set Headers for page https://www.thenetnaija.net/videos/series/11602-marvels-spider-man/season-3/episode-6/download"
time="2022-07-31T03:15:34+01:00" level=debug msg="Successfully retrieved body"
time="2022-07-31T03:15:34+01:00" level=debug msg="Retrieved Download Link https://www.thenetnaija.net/videos/series/11602-marvels-spider-man/season-3/episode-6/download\n"
time="2022-07-31T03:15:34+01:00" level=debug msg="Retrieving Download Link https://www.thenetnaija.net/videos/movies/7938-spider-man-into-the-spider-verse-2018\n"
time="2022-07-31T03:15:34+01:00" level=debug msg="Set Headers for page https://www.thenetnaija.net/videos/movies/7938-spider-man-into-the-spider-verse-2018"
time="2022-07-31T03:15:35+01:00" level=debug msg="Set Headers for page https://www.thenetnaija.net/videos/movies/7938-spider-man-into-the-spider-verse-2018"
time="2022-07-31T03:15:35+01:00" level=debug msg="Successfully retrieved body"
time="2022-07-31T03:15:35+01:00" level=debug msg="Retrieved Download Link https://www.thenetnaija.net/videos/movies/7938-spider-man-into-the-spider-verse-2018\n"
time="2022-07-31T03:15:35+01:00" level=info msg="Animation, Action, Adventure, Comedy, Family, Sci-FiRelease Date: Dec 14, 2018Stars: Shameik Moore, Jake Johnson, Hailee SteinfeldSource: Spider-Man.Into.the.Spider-Verse.2018.720p.WEB-DL.XviD.AC3-FGTLanguage: EnglishSubtitle Language: EnglishIMDB: https://www.imdb.com/title/tt4633694/"
time="2022-07-31T03:15:35+01:00" level=debug msg="Retrieving Download Link https://www.thenetnaija.net/videos/movies/7938-spider-man-into-the-spider-verse-2018/download\n"
time="2022-07-31T03:15:35+01:00" level=debug msg="Set Headers for page https://www.thenetnaija.net/videos/movies/7938-spider-man-into-the-spider-verse-2018/download"
time="2022-07-31T03:15:36+01:00" level=debug msg="Set Headers for page https://www.thenetnaija.net/videos/movies/7938-spider-man-into-the-spider-verse-2018/download"
time="2022-07-31T03:15:36+01:00" level=debug msg="Successfully retrieved body"
time="2022-07-31T03:15:36+01:00" level=debug msg="Retrieved Download Link https://www.thenetnaija.net/videos/movies/7938-spider-man-into-the-spider-verse-2018/download\n"
time="2022-07-31T03:15:36+01:00" level=debug msg="Retrieving Download Link https://www.thenetnaija.net/videos/series/11602-marvels-spider-man\n"
time="2022-07-31T03:15:36+01:00" level=debug msg="Set Headers for page https://www.thenetnaija.net/videos/series/11602-marvels-spider-man"
time="2022-07-31T03:15:37+01:00" level=debug msg="Set Headers for page https://www.thenetnaija.net/videos/series/11602-marvels-spider-man"
time="2022-07-31T03:15:37+01:00" level=debug msg="Successfully retrieved body"
time="2022-07-31T03:15:37+01:00" level=debug msg="Retrieved Download Link https://www.thenetnaija.net/videos/series/11602-marvels-spider-man\n"
time="2022-07-31T03:15:37+01:00" level=info msg="Animation, Action, Adventure, Family, Fantasy, Sci-FiIMDB: https://www.imdb.com/title/tt6135682/Premiere: Aug 19, 2017"
time="2022-07-31T03:15:37+01:00" level=debug msg="Retrieving Download Link https://www.thenetnaija.net/videos/series/11602-marvels-spider-man/download\n"
time="2022-07-31T03:15:37+01:00" level=debug msg="Set Headers for page https://www.thenetnaija.net/videos/series/11602-marvels-spider-man/download"
time="2022-07-31T03:15:38+01:00" level=debug msg="Set Headers for page https://www.thenetnaija.net/videos/series/11602-marvels-spider-man/download"
time="2022-07-31T03:15:38+01:00" level=debug msg="Successfully retrieved body"
time="2022-07-31T03:15:38+01:00" level=debug msg="Retrieved Download Link https://www.thenetnaija.net/videos/series/11602-marvels-spider-man/download\n"
time="2022-07-31T03:15:38+01:00" level=info msg="Animation, Action, Adventure, Family, Fantasy, Sci-FiIMDB: https://www.imdb.com/title/tt6135682/Premiere: Aug 19, 2017"
panic: runtime error: index out of range [2] with length 1
```

**Comment by MeNsaaH** [comment]
@derhnyel do you want to look into this?

**Comment by derhnyel** [comment]
yeah...cool

**Comment by deven96** [comment]
Fixed in #52
